<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Змейка</string>
  </property>
  <widget class="QLabel" name="NameGame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serto Malankara</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Супер змейка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Рекорды</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>92</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Цвет поля</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сложность</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Интеллект</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>92</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Сложность</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>107</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Легко</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>107</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Тяжело</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>107</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Средне</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Цвет змейки</string>
   </property>
   <widget class="QRadioButton" name="radioButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>107</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Зеленый</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>39</y>
      <width>107</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Красный</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
